apps/frontend/src/components/xpress/XpressProposalTable.tsx

```diff
@@ -289,7 +289,32 @@ const XpressProposalTable = ({ confirm }: { confirm: WithConfirmType }) => {
     t: TFunction<'translation', undefined>
   ) => [
     {
-      title: t('instrument'),
+      title: (
+        <>
+          <span>
+            {t('instrument')}
+            <Tooltip
+              title={
+                <span>
+                  <p>Tips: </p>
+                  <p>
+                    1.Change the status of a proposal to Under Review to enable
```

Comment:
Very minor but I see for the scientist comment we have a space after the `.` but not elsewhere